spring-xsuaa/pom.xml

```diff
@@ -167,7 +167,13 @@
 			<version>2.0.2</version>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
I don't understand why this new dependency is there. I could not find any new code that makes use of it? Was it transitively included before and now it is not anymore, so we need to list it explicitely as dependency?

Reply:
good catch 💯 that shouldn't be there, that's part of the JwtDecocer cache fix